File: opo.cpp
x=[]
while(True):
   a=input()
   if(a!='q'):
      x.append(int(a))
   else:
      break
y=list(filter(lambda a:a<=0 or a>120,x))
if(len(y)>0 or len(x)>20):
   print("INVALID INPUT")
else:
   sum=0
   for i in range of(len(x)):
       if(x[i]<17):
          sum+=200
       elif(x[i]>=17 and x[i]<=40):
          sum+=400
       else:
          sum+=300
   print("Total Income {} INR".format(sum)

File: opo.py
a = b = c = 0
d = []

while(1):
    n = input()
    if n.isdigit() == True:
        d.append(int(n))
    else:
        break

for i in d:
    if i > 0 and i < 17:
        a += 1
    if i >= 17 and i < 40:
        b += 1
    if i >= 40 and i < 120:
        c += 1
t = a*200+b*400+c*300
print("Total Income: ", t, "INR")
